GitHub.Collectors.Functions/GitHubFunctions.cs

```diff
@@ -498,10 +494,11 @@ private async Task ExecuteTrafficCollector(ExecutionContext executionContext, IL
                 InvocationId = executionContext.InvocationId.ToString(),
                 DequeueCount = dequeueCount,
             };
-
-            StatsTracker statsTracker = null;
-            bool success = false;
             ITelemetryClient telemetryClient = new GitHubApplicationInsightsTelemetryClient(this.telemetryClient, context);
+            CloudQueue trafficCloudQueue = await AzureHelpers.GetStorageQueueUsingMsiAsync("traffic", storageAccountNameEnvironmentVariable, telemetryClient).ConfigureAwait(false);
+            IQueue trafficQueue = new CloudQueueMsiWrapper(trafficCloudQueue, storageAccountNameEnvironmentVariable, telemetryClient);
+            StatsTracker statsTracker = null;
```

Comment:
Sometimes you wrap the CloudQueue into a CloudQueueMisWrapper and sometimes you use CloudQueue directly. Why? What's the difference? More specifically, can we use CloudQueueMsiWrapper everywhere?

Reply:
Modified code to use CloudQueueMsiWrapper consistently